interpreter/interpreter.py

```diff
@@ -116,11 +116,24 @@ def __init__(self):
     self.azure_api_version = None
     self.azure_deployment_name = None
     self.azure_api_type = "azure"
-    
-    # Get default system message
-    here = os.path.abspath(os.path.dirname(__file__))
-    with open(os.path.join(here, 'system_message.txt'), 'r') as f:
-      self.system_message = f.read().strip()
+
```

Comment:
If you think these changes are still relevant, I think they need to be moved to [`interpreter/utils/get_config.py`](https://github.com/KillianLucas/open-interpreter/blob/main/interpreter/utils/get_config.py) now that the code has been refactored.